packages/mobile/src/flags.ts

```diff
@@ -34,12 +34,6 @@ export const countryFeatures = {
   MOONPAY_DISABLED: {
```

Comment:
Is `MOONPAY_DISABLED` still being used anywhere?

Reply:
Yeah on the cash-out logic. I wanted to keep the scope of this PR focused but we will likely want to clean up the cash-out logic as well at a later time